src/ErrorProne.NET/ErrorProne.NET/Extensions/SyntaxNodeAnalysisContextExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Diagnostics.Contracts;
+using System.Text.RegularExpressions;
+using ErrorProne.NET.Common;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace ErrorProne.NET.Extensions {
+    public static class SyntaxNodeAnalysisContextExtensions {
+        public static INamedTypeSymbol GetClrType<T>(this SyntaxNodeAnalysisContext context) => context.SemanticModel.GetClrType(typeof(T));
```

Comment:
I think there is existing extensions that creates a named type. I would suggest to add this method there.